### PR TITLE
Make the Navbar responsive

### LINKED ISSUE #218: Make the navbar responsive
**Is your feature request related to a problem? Please describe.**
- The newly implemented navbar is not responsive for mobiles and other screen sizes.

**Describe the solution you'd like**
- Make the nav elements collapse when they reach a particular screen size. 

**Additional context**
(Refer to the link below for more information)
https://dev.to/devggaurav/let-s-build-a-responsive-navbar-and-hamburger-menu-using-html-css-and-javascript-4gci

### PR DESCRIPTION
**Pull request by Hasthikagirihagama**
## Purpose

The purpose of this PR is to fix #218

## Goals

To make the navigation bar responsive

## Approach

Made the navigation bar responsive according to the screen size

### Screenshots

Screenshot 2021-08-14 at 11 29 51
Screenshot 2021-08-15 at 20 47 51
Screenshot 2021-08-15 at 20 48 23

  
### Preview Link
https://hasthikagirihagama.github.io/sack-site/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -126,4 +146,5 @@ <h2 class="section-name">081 221 3652</h2>
 <script src="assets/js/core/popper.min.js"></script>
 <script src="./assets/js/core/bootstrap-material-design.min.js" type="text/javascript"></script>
 <script src="./assets/js/material-kit.js?v=2.0.7" type="text/javascript"></script>
+
```
Comment (kumuditha-udayanga) [inline index.html:149]: Remove this space

File: assets/css/custom.css [hunk 2]
```diff
@@ -6,6 +6,62 @@
...
+      position: fixed;
+      left: -100%;
+      top: 5rem;
+      flex-direction: column;
+      background-color: #fff;
+      width: 100%;
+      border-radius: 10px;
+      text-align: center;
+      transition: 0.3s;
+      box-shadow:
```
Comment (kumuditha-udayanga) [inline assets/css/custom.css:34]: Take this to a oneline

File: _includes/static/navbar.html [hunk 3]
```diff
@@ -31,6 +31,11 @@
               <li class="nav-item"><a href="javascript:;" class="nav-link">Link</a></li>
               <li class="nav-item"><a href="javascript:;" class="nav-link">Link</a></li>
             </ul>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+          </div>
```
Comment (kumuditha-udayanga) [inline _includes/static/navbar.html:38]: Format this part, the closing tag has an indentation issue.

**Review by kumuditha-udayanga — CHANGES_REQUESTED** [review]
Great Work! @Hasthikagirihagama 🥳, there are few changes.

**Comment by kumuditha-udayanga** [comment]
@jayasanka-sack, There is a issue with the contact number component and the search component in the navbar in mobile view, any suggestions.

File: _includes/static/navbar.html (hunk 3 above)
Comment (kumuditha-udayanga) [inline _includes/static/navbar.html:38]: Format this part, the closing tag has an indentation issue.

Comment (Hasthikagirihagama) [reply _includes/static/navbar.html:38]: yes what should I change in there?

File: assets/css/custom.css (hunk 2 above)
Comment (Hasthikagirihagama) [reply assets/css/custom.css:34]: It means should I remove this line ?

File: _includes/static/navbar.html [hunk 4]
```diff
@@ -23,8 +23,8 @@
           <div class="col-sm-8">
             <ul class="navbar-nav">
               <li class="active nav-item"><a href="javascript:;" class="nav-link">Admission
-                <div class="ripple-container"></div>
-              </a></li>
+                  <div class="ripple-container"></div>
+                </a></li>
```
Comment (kumuditha-udayanga) [inline _includes/static/navbar.html:27]: @Hasthikagirihagama,In this line `</a> </li>` is in the same line. Break those closing tags into two lines and align with their open tag.

File: _includes/static/navbar.html [hunk 5]
```diff
@@ -23,8 +23,8 @@
           <div class="col-sm-8">
             <ul class="navbar-nav">
               <li class="active nav-item"><a href="javascript:;" class="nav-link">Admission
```
Comment (kumuditha-udayanga) [inline _includes/static/navbar.html:25]: In this line too, Break `<li>` and `<a>` break them into two lines. Always format the code before adding them. (Only the part that you add or change, not the whole code)

**Review by kumuditha-udayanga — CHANGES_REQUESTED** [review]
Always, select the part of the code you changed, and format it using (Shift + Option + F). Then all indentation issues will be automatically be fixed.

**Comment by Hasthikagirihagama** [comment]
I think now the errors are fixed

**Comment by kumuditha-udayanga** [comment]
@Hasthikagirihagama, Great work!, there's a small issue try to fix it
https://user-images.githubusercontent.com/27630091/132743160-4dd48aba-c127-48b4-90c3-2832a468ca00.mov

**Comment by jayasanka-sack** [comment]
@kumuditha-udayanga Could you please review this again?

**Review by kumuditha-udayanga — APPROVED** [review]
Thank you @Hasthikagirihagama, amazing work!

**Pull request merged**